### DISCUSSION TITLE
Apex Tests should be grouped by Package

### DISCUSSION THREAD

**Discussion by ntotten** [forcedotcom/salesforcedx-vscode#5743] [Ideas]
Currently, all tests in a project are listed in the Apex Test view in alphabetical order. If you have multiple packages in your solution this list can be very long. When a workspace has more than one package (as specified in sfdx-project.json) the tests should be grouped by Package.

There would be a new right-click run option on a package: Run all Tests in Package.

**Comment by mshanemc** [comment]
they are now!
